packages/features/landing_page_screen/lib/src/landing_page.dart

```diff
@@ -232,9 +232,10 @@ class _ProjectPreview extends StatelessWidget {
           child: latestModifiedProject == null
               ? null
               : ProjectOverflowMenu(
-                  key: const Key('ProjectOverflowMenu Key0'),
-                  project: latestModifiedProject!,
-                ),
+            key: const Key('ProjectOverflowMenu Key0'),
+            project: latestModifiedProject!,
+            iconColor: const Color(0xFF191C1D),
```

Comment:
This is from lightColorScheme only 
"onBackground: Color(0xFF191C1D),"
I thought of using the code directly instead of importing color_scheme for one colour 
If you want, I can do it other way.